client/src/components/RatingsAndReviews/ReviewHelpfulness.jsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
```

Comment:
sx-a11y/click-events-have-key-events and jsx-a11y/no-noninteractive-element-interactions not necessary